src/components/profile/ChangePasswordForm.jsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Alert, TextInputField } from "evergreen-ui";
+import Button from "../ui/button/Button";
+import Spinner from "../ui/loader/Spinner"
+
+
+import { connect } from "react-redux";
+import { getChangePasswordInfos } from "../../store/modules/user/selectors";
+import { 
+    UPDATE_USER_PASSWORD_PENDING,
+    UPDATE_USER_PASSWORD_SUCCESS, 
+    UPDATE_USER_PASSWORD_ERROR, } from "../../store/modules/user/constants"
+
+import { changeUserPassword } from "../../store/modules/user/user";
+
+
+class ChangePasswordForm extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            oldPwd: "",
+            newPwd: "",
+            newPwdConf: ""
+        };
+        this.onChangePasswordSubmitted = this.onChangePasswordSubmitted.bind(this);
+    }
+
+    onChangePasswordSubmitted(event) {
```

Comment:
If you use an arrow ES6 function you don't need to bind it 😃 It will be done for you

```suggestion
    }

    onChangePasswordSubmitted = (event) => {
```